### ISSUE TITLE
Volume Binding not working in VeryCrypt drives

### ISSUE THREAD

**Issue by SalkinD** [docker/for-win#12340]
I am trying to get a docker app working with using a database provided on an external encrypted drive. This causes weird behaviour.

  - [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 413ADC87-792F-4AD6-9851-F4A83C523C92/20211103145909

### Actual behavior

- Bound Volume not working on encrypted drive using VeraCrypt. No error. Absolutely weird behaviour of bound drives. (Folders differ, when overriding an existing folder old content is shown, no sync in both directions)

### Expected behavior

Bound volume working.

It may be possible to reproduce this error:
 - Use WSL2 / Windows 10
 - Create an VeraCrypt Volume and mount
 - bind any folder on this Volume to a docker container

EDIT: Start Docker before mounting Volume and opposite the behaviour changes in my case but in both cases buggy

### Information

  - Windows Version: 10
  - WSL2

**Comment by JKocerka** [comment]
@SalkinD - did you try to mount a normal (non-encrypted) folder?

**Comment by thenyak** [comment]
@SalkinD I just experienced the same thing after I had it working previously. Then I realized the difference was when I had it working I didn't have Docker Desktop starting up automatically. I was mounting my VeraCrypt volume first then starting Docker Desktop. Solutions appear to be:

1. If starting Docker Desktop automatically -> restart Docker Desktop after mounting VeraCrypt volume
2. If starting Docker Desktop manually -> mount VeraCrypt volume first then start Docker Desktop

The really strange thing to me is, before I figured this out, the volume was persisting but I could not find the persisted files anywhere on my VeraCrypt drive. Makes me wonder what VeraCrypt is doing and where/how it actually got persisted.
